src/todo.spec.js

```diff
@@ -312,3 +312,19 @@ describe('findByStatus', () => {
       .toThrow("No todos found with status: done");
   });
 });
+
+describe('findByTitle', () => {
+  it('should find the titles that contains the given text', () => {
+    const mockStore = createMockStore([
+      { id: 1, title: 'Todo abc', done: false },
+      { id: 2, title: 'Todo def', done: true }
+    ])
+    
+    const expected = [
+      { id: 2, title: 'Todo def', done: true }
+      ];
+    const current = findByTitle(mockStore, ['def']);
+    expect(current).toStrictEqual(expected);
+  });
+  
```

Comment:
You can add a new test case here which is testing if you add todo with all capital letter as parameter like that ---> TODO